src/FactSystem/FactControls/FactStepper.qml

```diff
@@ -0,0 +1,130 @@
+/****************************************************************************
+ *
+ *   (c) 2009-2016 QGROUNDCONTROL PROJECT <http://www.qgroundcontrol.org>
+ *
+ * QGroundControl is licensed according to the terms in the file
+ * COPYING.md in the root of the source code directory.
+ *
+ ****************************************************************************/
+
+import QtQuick 2.5
+import QtQuick.Controls 1.2
+import QtQuick.Controls.Styles 1.2
+import QtQuick.Dialogs         1.2
+
+import QGroundControl.ScreenTools   1.0
+import QGroundControl.FactSystem 1.0
+import QGroundControl.Palette 1.0
+import QGroundControl.Controls 1.0
+
+Item {
+    id: stepper
+    property bool enabled: true
+    property alias fact: factInput.fact
+    property alias showHelp: factInput.showHelp
+    property alias showUnits: factInput.showUnits
+
+    property string _valueText:          fact ? fact.valueString : "N/A"
+    property double _factValue:          Number(_valueText)
+
+    property double minimumValue:       !fact || isNaN(fact.min) ? 0 : fact.min
```

Comment:
I followed analogy with `isNaN(fact.increment)` from the slider panel.
And they are not always set. It might be just bad data, but I don't own these values, so I would prefer to check. My default min/max might be not good tho.